docs/memtierd/README.md

```diff
@@ -0,0 +1,77 @@
+# Memtierd NRI plugin
+
+An easy way to start using [Memtierd](https://github.com/intel/memtierd/tree/main) as a memory manager in your Kubernetes cluster.
+
+## Prerequisities
+- NRI enabled on your container runtime
+
+## Making your own image
+
+The Memtierd binary needs to built first. To do so, follow the instructions found [here](https://github.com/intel/memtierd/tree/main/cmd/memtierd).
```

Comment:
I think you should do a `go install github.com/intel/memtierd/cmd/memtierd@latest` from the dockerfile, then copy the binary from the golang installed path to /bin... if memtierd can be compiled with a simple 'go build/go install'. If not, then cloning the repo in the Dockerfile then building with make should work.